frontend/packages/console-app/src/components/quick-starts/QuickStartController.tsx

```diff
@@ -65,13 +74,15 @@ const QuickStartController: React.FC<QuickStartControllerProps> = ({ quickStart
         onTaskSelect={handleTaskSelect}
         onTaskReview={handleTaskStatusChange}
         onQuickStartChange={handleQuickStartChange}
+        ref={ref}
```

Comment:
Pass the incoming prop straight down to the content child:
```suggestion
        ref={contentRef}
```